R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -10,6 +10,23 @@ vignette: >
   \usepackage[utf8]{inputenc}
 ---
 
+<!--
```

Comment:
FWIW the ASF policy statement talks about source files only: http://www.apache.org/legal/release-policy.html#license-headers
Is this source for docs? maybe so. I think it's fine as-is then.